pkg/apis/kops/validation/validation_test.go

```diff
@@ -549,6 +549,24 @@ func Test_Validate_Calico(t *testing.T) {
 				Etcd: kops.EtcdClusterSpec{},
 			},
 		},
+		{
+			Input: caliInput{
+				Calico: &kops.CalicoNetworkingSpec{
+					MajorVersion:     "v3",
+					WireguardEnabled: true,
+				},
+				Etcd: kops.EtcdClusterSpec{},
+			},
+		},
+		{
+			Input: caliInput{
+				Calico: &kops.CalicoNetworkingSpec{
+					WireguardEnabled: true,
+				},
+				Etcd: kops.EtcdClusterSpec{},
+			},
+			ExpectedErrors: []string{"Invalid value::calico.wireguardEnabled"},
+		},
```

Comment:
`v3` is the default now and `v2` will be removed in kops 1.20, so we can skip this.
```suggestion
```